pepper-apis/dsm-core/src/main/resources/liquibase/juniper-ourheart/PEPPER-1057-our_heart_setup.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="pegah" id="new-juniper-study-instance-role">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT COUNT(*) FROM instance_role WHERE name='juniper_study';
+            </sqlCheck>
+        </preConditions>
+        <insert tableName="instance_role">
+            <column name="name" value="juniper_study"/>
+        </insert>
+    </changeSet>
+    <changeSet id="our-health-ddp-instance-insert" author="pegah">
+        <insert tableName="ddp_instance">
+            <column name="instance_name" value="juniper-ourheart"/>
+            <column name="study_guid" value="juniper-ourheart"/>
+            <column name="display_name" value="Our Health"/>
```

Comment:
no it's our heart! I should have made a comment about it sorry!

Basically OurHealth is the group name so they _might_ want to run other studies, so because this was their heart study the juniper side chose the ourheart as their study name

We are only choosing the display name to be our health for now because GP knows the study as named "our health".